priv/static-template/wasm/popcorn.js

```diff
@@ -249,6 +249,10 @@ export class Popcorn {
     this._iframe = null;
     this._mountPromise = null;
     this._listenerRef = null;
+    for (const [_id, callData] of this._calls) {
+      const durationMs = performance.now() - callData.startTimeMs;
+      callData.reject({ error: "PopcornDeinit", durationMs })
```

Comment:
Please use `new Error("Call cancelled due to instance deinit")` or similar.

Reply:
Reject throws an exception anyways, we can alternatively subclass `Error` to add `PopcornCancellationError` or something.

Reply:
done